src/approaches/base_approach.py

```diff
@@ -9,6 +9,7 @@
     ParameterizedOption, Action, Dataset, Metrics, InteractionRequest, \
```

Comment:
Could you add an example command line and attach the resulting video to this PR?